### DISCUSSION TITLE
Remote desktop interaction

### DISCUSSION THREAD

**Discussion by Thangta03** [BambooEngine/ibus-bamboo#473] [General]
Em dùng Fedora 39 gặp lỗi liên tục yêu cầu bật Remote desktop interaction. Sau khi tắt bàn phím đi thì không bị nữa. Em khá nhát nên không dám thử nhiều, mong cao nhân vào giải thích giúp

**Comment by t1ld3x** [comment]
bạn có thể chụp lại ảnh màn hình để mô tả rõ hơn không?

**Comment by boothedev** [comment]
Mình cũng bị từ Fedora 39 (Gnome 45). Mình tìm thấy #430 và tắt tính năng "Bắt sự kiện chuột" trong cài đặt của ibus-bamboo thì không bị nữa. Mình nghĩ nên có lưu ý cho vấn đề này.

**Reply by HieuNGN** [reply]
Mình nghĩ không phải do Bắt sự kiện chuột mà lỗi gì đó của Gnome. Đã search GG và không còn bị.

**Comment by HieuNGN** [comment]
Lỗi này do Gnome đó bạn, lên google search sẽ ra cách xử lí.
